libopenfpga/libfpgaaesencryption/src/AESfunctions.h

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+using namespace std;

```

Comment:
Using the std namespace is not recommended. This is a planted bomb. When it is included in any source files, it will force a namespace for any functions which do not belong to this file.
I suggest to use 
```
namespace openfpga {
  <functions or variable...>
}
```
You can change to other name spaces